kube-flagd-proxy/cmd/root.go

```diff
@@ -0,0 +1,76 @@
+package cmd
```

Comment:
Needs cleanup - several uses of `flagd` instead of `<NAME>`. Yes we need a NAME for this component. 

@toddbaert  & @beeme1mr  any thoughts?

Reply:
Yeah this was a direct copy and paste from the Flagg cmd, I agree this needs fixing, IMO im happy to call it `Kubernetes-flagd-proxy` here to match the other references